superset/db_engine_specs/presto.py

```diff
@@ -872,7 +872,7 @@ def expand_data(  # pylint: disable=too-many-locals
                     values = row.get(name) or []
                     if isinstance(values, str):
                         row[name] = values = cast(List[Any], destringify(values))
-                    for value, col in zip(values, expanded):
+                    for value, col in zip(values or [], expanded):
```

Comment:
As the destringified values may be `None`, we should probably also update line 874 to reflect this by updating the cast to more accurately reflect the expected return value:
```python
row[name] = values = cast(Optional[List[Any]], destringify(values))
```
With this change mypy would see that `values` may be `None`. Alternatively we may want to move the `or []` up one line, but in this case the return value will change (not sure which is better).

Reply:
You're right. Thank you. I changed it as per your first suggestion and force pushed the updated commit. I had to split the statement into two because of line length limit.